signing_service/signing_service.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash -e
 
-python3 src/signing_service.py "$@"
+python3 -m signing_service.signing_service "$@"
```

Comment:
It looks like this change is in a wrong commit.

Reply:
Why ? The commit name is `Add setup.py to SigningService **and update of running script**`

Reply:
Because it looks like a part of the file reorganization that's in the next commit. I thought that you mistakenly included it in the wrong commit. Does this work even without the reorganization?

If it's not meant to be a part of that commit then I think it should be a separate commit. The fact that you need to use _and_ in the description indicates that you're doing two different things in the same commit.

Reply:
Right, it won't work before file reorganization, so I will move it to that commit. :+1: done